examples/image-streams/image-streams-centos7.json

```diff
@@ -22,6 +22,7 @@
               "description": "Build and run Ruby 2.0 applications",
               "iconClass": "icon-ruby",
               "tags": "builder,ruby",
+              "supports": "ruby:2.0,ruby",
```

Comment:
Ok, be explicit *AND* do both checks.  I think the priority order should be "if you say ruby:2.0, we'll first check for IS ruby and tag 2.0, then search IS ruby for tag 2.0, then search other IS for them".  Is that what you're implementing (and by order I really mean weight)?